### PR TITLE
error handling example should not rescue Exception

### PR DESCRIPTION
**Pull request by ckhall**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ it can't connect to the server a `Redis::CannotConnectError` error will be raise
 ```ruby
 begin
   redis.ping
-rescue Exception => e
+rescue => e
```
Comment (drm2) [inline README.md]: This is invalid. You should probably use `StandardError`:

```ruby
rescue StandardError => e
```

Comment (ckhall) [reply README.md]: yes, wrt linters, makes sense 👍

**Comment by drm2** [comment]
@ckhall looks good. You might want to squash your 2 commits into 1 so there isn't a dead commit in the history.

**Comment by ckhall** [comment]
@drm2 thanks for the tip.  done.

**Comment by mperham** [comment]
LGTM

**Pull request merged**